components/contextmenu/overview.md

```diff
@@ -14,14 +14,14 @@ The <a href="https://www.telerik.com/blazor-ui/context-menu" target="_blank">Bla
 
 In addition to built-in [navigation capabilities]({%slug contextmenu-navigation%}), you can browse through the items and their children, define [templates]({%slug contextmenu-templates-overview%}) for the individual nodes, render text and icons/images, and respond to [events]({%slug contextmenu-events%}).
 
-#### To use a Telerik Context Menu for Blazor:
+## Creating Context Menu
 
-1. add the `TelerikContextMenu` tag and set its `Selector` parameter to a CSS selector that will match the element(s) you want to attach the context menu to.
-1. provide a collection of models to its `Data` property (read more in the [Data Binding article]({%slug contextmenu-data-binding-overview%}))
-1. match the fields in the models with the binding schema for the nodes
-1. handle the `OnClick` [event]({%slug contextmenu-events%}) to respond to the user action
+1. Add the `TelerikContextMenu` tag and set its `Selector` parameter to a CSS selector that will match the element(s) you want to attach the context menu to.
+1. Provide a collection of models to its `Data` property (read more in the [Data Binding article]({%slug contextmenu-data-binding-overview%}))
+1. Match the fields in the models with the binding schema for the nodes
```

Comment:
Consider something like point 2 here - https://docs.telerik.com/blazor-ui/components/treeview/overview#creating-blazor-treeview

I.e. use default property names, skip the configuration and just mention there is a way to customize and map the property names.